src/d/actor/d_a_obj_eskban.cpp

```diff
@@ -3,105 +3,451 @@
+        }
+        dComIfGp_event_onEventFlag(DESTROY_EVT_FLAG);
+        /* fallthrough */
+    case ST_VIBRATING: /* 0x388 */
+        if (mRemainingVibration > 0) {
+            mRemainingVibration--;
+            if (mRemainingVibration == DESTROY_VIBRATION_SHOCK_FRAME_IDX) {
+                dComIfGp_getVibration().StartShock(6, 0x4, cXyz(0, 1, 0));
+            }
+        } else if (!fopAcM_isSwitch(this, param_get_swSave())) {
+            fopAcM_onSwitch(this, param_get_swSave());
+            dComIfGp_getVibration().StopQuake(-1);
+            dComIfGp_getVibration().StartQuake(4, ~0x20, cXyz(0, 1, 0));
+        }
+        if (mRemainingSmokeAnm > 0) {
+            mRemainingSmokeAnm--;
+            break;
+        }
+        if (dComIfGp_evmng_endCheck("Eskban") && (this->M_smoke->isEnd())) {
+            dComIfGp_getVibration().StopQuake(-1);
+            fopAcM_delete(this);
+            dComIfGp_event_onEventFlag(DESTROY_EVT_FLAG);
+        }
+    }
+    set_mtx();
+    *pMtx = &M_tmp_mtx;
+    return TRUE;
 }
 
 /* 00001400-000014B4       .text Draw__Q211daObjEskban5Act_cFv */
 BOOL daObjEskban::Act_c::Draw() {
-    /* Nonmatching */
+    g_env_light.settingTevStruct(TEV_TYPE_BG0, &this->current.pos, &this->tevStr);
+    g_env_light.setLightTevColorType(mpModel, &this->tevStr);
+    if (!mIsVisible) {
+        return TRUE;
+    }
+    dComIfGd_setListBG();
+    mDoExt_modelUpdateDL(mpModel);
+    dComIfGd_setList();
+    return TRUE;
 }
 
 namespace daObjEskban {
 namespace {
-/* 000014B4-000014D4       .text Mthd_Create__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv */
-void Mthd_Create(void*) {
-    /* Nonmatching */
+/* 000014B4-000014D4       .text Mthd_Create__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv
+ */
+void Mthd_Create(void* i_this) {
+    ((Act_c*)i_this)->Mthd_Create();
 }
 
-/* 000014D4-000014F4       .text Mthd_Delete__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv */
-void Mthd_Delete(void*) {
-    /* Nonmatching */
+/* 000014D4-000014F4       .text Mthd_Delete__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv
+ */
+void Mthd_Delete(void* i_this) {
+    ((Act_c*)i_this)->Mthd_Delete();
 }
 
-/* 000014F4-00001514       .text Mthd_Execute__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv */
-void Mthd_Execute(void*) {
-    /* Nonmatching */
+/* 000014F4-00001514       .text Mthd_Execute__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv
+ */
+void Mthd_Execute(void* i_this) {
+    ((Act_c*)i_this)->MoveBGExecute();
 }
 
 /* 00001514-00001540       .text Mthd_Draw__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv */
-void Mthd_Draw(void*) {
-    /* Nonmatching */
+void Mthd_Draw(void* i_this) {
+    ((Act_c*)i_this)->MoveBGDraw();
 }
 
-/* 00001540-0000156C       .text Mthd_IsDelete__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv */
-void Mthd_IsDelete(void*) {
-    /* Nonmatching */
+/* 00001540-0000156C       .text Mthd_IsDelete__Q211daObjEskban28@unnamed@d_a_obj_eskban_cpp@FPv
+ */
```

Comment:
can you disable the autoformatter that's randomly adding/removing linebreaks in autogenerated lines of code like this? I often do regex searches on these and it's easier if they don't get modified from how they were originally

Reply:
I'll disable it for this block with a comment (`// clang-format off`, I see there are already a couple other instances of it in the repo), but I think it could be globally deactivated for these comments that start with an offset range by adding it to [CommentPragmas](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#commentpragmas)) in the `.clang-format` https://github.com/zeldaret/tww/blob/37d09d16577fdefb9bafa50dbb3da07980bc4e62/.clang-format#L39
```diff
-CommentPragmas:  '^ (IWYU pragma:|NOLINT)'
+CommentPragmas:  '^ (IWYU pragma:|NOLINT|\d{8}-\d{8}\b)'
```

Reply:
I think the clang stuff was copied from TP decomp, I don't think we use that here at all

Reply:
I removed it.

Reply:
ah ok, I removed the changes it made to the skeleton and disabled it locally then